lego/apps/users/models.py

```diff
@@ -441,8 +441,32 @@ def email_address(self):
     def profile_picture(self, value):
         self.picture = value
 
+    def verify_student(self, feide_groups) -> bool:
+        if self.is_verified_student():
+            return True
+
+        if self.has_grade_group:
+            self.student_verification_status = True
+            self.save()
+            return True
```

Comment:
Does this mean anyone who is already verified and in a grade group today, can verify without any valid FEIDE group?
I feel like we should save who is verified with FEIDE, and who is verified with the old system in case we want to verify everyone through FEIDE later.

Reply:
Yes. This is what we discussed on slack and the solution I've implemented here seemed to be the consensus.

About differentiating the two, I see your point and thought about it. But I feel like we would just add extra logic to replace later. Sure, it would be extra work for the users who have authenticated already if we require everyone to re-verify. But it literally takes two seconds, so I don't _really_ feel like it's that big of a deal? The only thing is that anyone who might have been manually verified will need to do that again. But that would just require us to keep a list of those that will apply to.

I am open to suggestions though. But I'd like the solution to not include too many (rather 0) changes to the DB and lots of logic (event registrations etc.)

Reply:
Ok. I guess that's fine. We'll just keep a list of the manually verified users